config/configuration.json

```diff
@@ -711,6 +711,17 @@
       "difficulty": "NORMAL"
     }
   },
+  "RoundaboutMissingTagCheck": {
+    "enabled": true,
```

Comment:
Looks like the config is missing the "threshold.degrees" keys.

Reply:
thanks! added.